Bootloader/main.c

```diff
@@ -219,7 +223,11 @@ void init_usb_bootloader( int config )
 	dfu_init( setup_read, setup_write, finish_write, &dfu_ctx );
 
 	// Make sure SysTick counter is disabled (dfu has issues otherwise)
```

Comment:
Just a note here, I noticed that interrupts had some odd behaviour with dfu. This may be a bug in the bootloader code, or perhaps the tight timing constraints of writing to flash (or getting interrupts while flash is trying to be written). This is important to note because all of the delay code requires the systick now (which needs interrupts).